src/Plugin/KeyInput/ApigeeAuthKeyInput.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 /**
- * Copyright 2018 Google Inc.
+ * Copyright 2020 Google Inc.
```

Comment:
As far as I remember, the data here is when the file was created and not when the file was last modified. So it should not change #fixme.